components/eam/bld/configure

```diff
@@ -1510,9 +1511,19 @@ else {
 
     }
 
-    if (defined $opts{'use_MMF_VT'}) {
-        $nadv += 2; # add tracers for CRM t and q variance transport
-        if ($print>=2) { print "Advected variance constituent added for MMF: 2$eol"; }
+    ### advected tracers for MMF
+    if (defined $opts{'use_MMF'}) {
+        $nadv += 8;
```

Comment:
Before we had a note that stated what these extra tracers correspond to. Could you add a note here to explain what the 8 added tracers are for?

Reply:
I'll add a note for this and merge this block with the one above.

Reply:
done.